test/nodes/test_file_converter.py

```diff
@@ -65,6 +65,7 @@ def test_pdftoppm_command_format():
     ), 'Your installation of poppler is incompatible with Haystack. Try installing via "conda install -c conda-forge poppler"'
 
 
+@pytest.mark.pdf
```

Comment:
This and the following markers need to be changed.
```suggestion
@pytest.mark.unit
```